Import sqlalchemy-enum-tables

ttung opened the pull request:
### Description
Importing this repo because we need to make changes to it.  I am working to clear this with the open source team @ CZI.

We are importing from https://git.heptacle.fr/neshiraini/sqlalchemy-enum-tables @ f2d832ad78aeac2acd6c0a6e22a6d43049a2c409

### Test plan
none

ttung commented:
Ok Shawn Liu approved this. :)

phoenixAja commented on third-party/sqlalchemy-enum-tables/.gitlab-ci.yml:
```diff
@@ -0,0 +1,69 @@
+stages:
```
will this still be triggered if it's named `gitlab` instead of `github`?

ttung replied:
No, it won't but there really aren't any tests worth running.

Commit:
Import sqlalchemy-enum-tables

### Description
Importing this repo because we need to make changes to it.

We are importing from https://git.heptacle.fr/neshiraini/sqlalchemy-enum-tables @ f2d832ad78aeac2acd6c0a6e22a6d43049a2c409

### Test plan
none
